Library/Homebrew/livecheck/strategy/git.rb

```diff
@@ -86,6 +86,21 @@ def self.find_versions(url, regex = nil)
 
           tags_only_debian = tags_data[:tags].all? { |tag| tag.start_with?("debian/") }
 
+          if block
```

Comment:
```suggestion
          if block_given?
```

Reply:
In a method with an explicit block parameter, `brew style` prefers that you check the block argument (`block` in this case) instead of using `block_given?`. This change produces the following style error:

```
/usr/local/Homebrew/Library/Homebrew/livecheck/strategy/git.rb:89:14: C: Check block argument explicitly instead of using block_given?.
          if block_given?
             ^^^^^^^^^^^^
```